frame/composable-traits/src/dex.rs

```diff
@@ -168,22 +168,25 @@ pub trait DexRouter<AccountId, AssetId, PoolId, Balance, MaxHops> {
 	) -> Result<(), DispatchError>;
 	/// If route exist return `Some(Vec<PoolId>)`, else `None`.
 	fn get_route(asset_pair: CurrencyPair<AssetId>) -> Option<Vec<DexRouteNode<PoolId>>>;
-	/// Exchange `dx` of `base` asset of `asset_pair` with associated route.
+	/// Exchange `amount` of `quote` asset of `asset_pair` with associated route.
 	fn exchange(
 		who: &AccountId,
 		asset_pair: CurrencyPair<AssetId>,
-		dx: Balance,
+		amount: Balance,
+		min_receive: Balance,
 	) -> Result<Balance, DispatchError>;
-	/// Sell `amount` of `base` asset of asset_pair with associated route.
+	/// Sell `amount` of `quote` asset of asset_pair with associated route.
```

Comment:
it was right. base usually afaik is something like unsalbe BTC. while quote usually USD. so you sell 100 BTC for >=10000 USD 
buy 100 BTC for <= 10000 USD in quite